### PR TITLE
feat: dockefile for ncr and publish on ghcr

### PR DESCRIPTION
**Pull request by albertolerda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,6 +122,14 @@ make
 
 Then point your browser to the [http://localhost:3000/docs](http://localhost:3000/docs) to see all the exposed endpoints
 
+
+## 🐋 Docker
+You can start it using docker, just have to mount you static file directory
+```
+docker pull ghcr.io/albertolerda/didroom_microservices:latest
```
Comment (puria) [inline README.md:129]: fix the address

File: credential_issuer/health.keys.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "timestamp_srv": "http://apiroom.net:3312"
```
Comment (puria) [inline credential_issuer/health.keys.json:2]: ahaha this is not reliable at all ;p do a query on the server itself, a localhost... so you check that is tautologically doing a self-assesment plz!

File: credential_issuer/health.schema.json
Comment (puria) [inline credential_issuer/health.schema.json:1]: The schema also is relaly not needed, I guess no-intended use via GUI...

**Review by puria — CHANGES_REQUESTED** [review]

File: credential_issuer/health.schema.json
Comment (albertolerda) [reply credential_issuer/health.schema.json:1]: without the schema, the service didn't start...

**Review by puria — APPROVED** [review]

**Review by puria — APPROVED** [review]

**Pull request merged**